### PR TITLE
Start CSI proxy as service

### PR DESCRIPTION
**Pull request by jingxu97**
Now CSI proxy can support starting as service. Update node startup
script to create and start it as service

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by jingxu97** [comment]
@pjh could you help approve?

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 1]
```diff
@@ -420,8 +420,11 @@ function DownloadAndInstall-CSIProxyBinaries {
 # TODO(jingxu97): Make csi-proxy.exe as a service similar to kubelet.exe
```
Comment (pjh) [inline cluster/gce/windows/k8s-node-setup.psm1:420]: You can remove this TODO now :)

**Review by pjh — APPROVED** [review]

File: cluster/gce/windows/k8s-node-setup.psm1 (hunk 1 above)
Comment (jingxu97) [reply cluster/gce/windows/k8s-node-setup.psm1:420]: sure, removed. thanks!

**Pull request merged**